applications/MeshingApplication/custom_python/add_meshers_to_python.cpp

```diff
@@ -118,6 +119,19 @@ void TriRegenerateMesh(TriGenPFEMModeler& Mesher, char* ElementName, char* Condi
                           KratosComponents<Condition>::Get(ConditionName),node_erase, rem_nodes, add_nodes, alpha_shape, h_factor	);
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////
+//											//
+//				ADAPTIVE 2D MESHER specifically for GLASS FORMING 	//
+//											//
+//////////////////////////////////////////////////////////////////////////////////////////
+
+void TriRegenerateMeshGLASS(TriGenGLASSModeler& Mesher, char* ElementName, char* ConditionName, ModelPart& model_part,NodeEraseProcess& node_erase, bool rem_nodes, bool add_nodes, double alpha_shape, double h_factor )
```

Comment:
Just a comment, the input variables in a method should be in CammelCase https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide

Reply:
didnt get it. could you please indicate which of the variables doesnt follow this format?

Reply:
`model_part` -> `rModelPart`
`node_erase` -> `TheNodeEraseProcess`
and so on

Reply:
okay, cool! thanks!